chess/src/Components/Networking.js

```diff
@@ -1,4 +1,5 @@
-const API_URL = process.env.REACT_APP_API_URL;
+// const API_URL = process.env.REACT_APP_API_URL;
```

Comment:
done need this comment

Reply:
This is what it was before, I'm keeping it commented out incase this change doesn't work